c10/core/ScalarType.cpp

```diff
@@ -222,6 +222,8 @@ std::pair<std::string, std::string> getDtypeNames(c10::ScalarType scalarType) {
       return std::make_pair("float8_e5m2fnuz", "");
     case c10::ScalarType::Float8_e4m3fnuz:
       return std::make_pair("float8_e4m3fnuz", "");
+    case c10::ScalarType::Float8_e8m0fnu:
+      return std::make_pair("float8_e8m0fnu", "");
```

Comment:
This would be easily to macro-ify and apply for all float8 types.

Reply:
tracking in https://github.com/pytorch/pytorch/issues/146647